src/backend/marsha/core/models/video.py

```diff
@@ -121,7 +121,8 @@ def is_ready_to_show(self):
         `uploaded_on` field but it is necessary for conveniency and clarity in the client.
         """
         return (
-            self.uploaded_on is not None and self.upload_state != HARVESTED
+            self.uploaded_on is not None
+            and self.upload_state not in [HARVESTED, DELETED]
```

Comment:
do we really need this DELETED value for the upload_state or is it redundant with deleting the video? Or is it a different state?

Reply:
We can really delete the video yes but the instructor will don't find the video any more and the LTI block in the LMS will show an error message.

Reply:
> the instructor will don't find the video any more

So what happens for the instructor with this DELETE state?

> the LTI block in the LMS will show an error message

Or wille it create a new video? :thinking:

Reply:
> So what happens for the instructor with this DELETE state?

A message saying that the video is not available anymore is display

> Or wille it create a new video? thinking

We need to test it. Does safe delete prevent to create a video with an existing UUID ? Do we want to create this new video, I think no.

Reply:
ok then it's probably something different. Leave it like this :shrug:

Reply:
After thoughts: it should be called "ENDED" and display a message to the viewer that this live has ended. The teacher may get more explanation about the fact that the delay to download the file has ended too?

Reply:
Ok i will create a new PR for this modifications.